core/src/main/java/org/stellar/anchor/sep31/Refunds.java

```diff
@@ -15,4 +16,23 @@ public interface Refunds {
   List<RefundPayment> getRefundPayments();
 
   void setRefundPayments(List<RefundPayment> refundPayments);
+
+  default org.stellar.anchor.api.shared.Refund toPlatformApiRefund(String assetName) {
+    // build payments
+    org.stellar.anchor.api.shared.RefundPayment[] payments = null;
```

Comment:
nit: 💡 add static import?

Reply:
The thing is we have different objects with the same name:
* org.stellar.anchor.sep31.RefundPayment
* org.stellar.anchor.api.shared.RefundPayment